src/widget.cpp

```diff
@@ -513,6 +513,38 @@ namespace gcn
         }
     }
 
+    void Widget::setGlobalBaseColor(Color color)
+    {
+        for (Widget* w : mWidgetInstances)
+        {
+            w->setBaseColor(color);
```

Comment:
equivalent function for font doesn't overwrite, we should be coherent.
or rename the function?

Reply:
Don't all the widgets already _have_ a default color though? If we checked for that, they would never get the global color.

Besides, I think this is useful in cases where you want to apply a global color to all the widgets, regardless of their current one. Think of having different GUI "themes" for instance, and you want to switch from one to another. All widgets would currently have a color, but you want it changed to another.

The font case is a bit different. It too sets the global Font right away, then triggers a `fontChanged` event if the current font of the widget is not set. I don't think we have something similar for colors however.